packages/nestjs/README.md

```diff
@@ -0,0 +1,376 @@
+> Full user management with Accounts.JS and Nest.js in minutes.
+
+# Basic Usage
+
+`npm i @nb/nestjs-accountsjs`
+
+> note: currently I only have this on my personal NPM registry. Make an issue to remind me to make it public if you can't install it with `npm i @nb/nestjs-accountsjs --registry npm.nickbolles.com`
+
+### [Basic Config](./examples/basic-value-opts)
+
+app.module.ts
+
+```typescript
+import { Module } from '@nestjs/common';
+import { Mongo } from '@accounts/mongo';
+import { AccountsPassword } from '@accounts/password';
+import { AccountsJsModule } from '@nb/nestjs-accountsjs';
+
+@Module({
+  imports: [
+    AccountsJsModule.register({
+      serverOptions: {

```

Comment:
would this setup second express server in parallel to nest?

Reply:
this example is setting up an accounts server. I don't know why you'd have express outside of nest, nest just wraps express. Look about 15 lines down for the "with Accounts Server Instance" section if you for some reason already have the accounts server instance created.

Reply:
yeah i suppose it doen not setup anything in parallel just. but the name is misleading , what doest this server serves then. i'd rather called it `options`

Reply:
I guess, but the whole argument to register is options already. and inside of that there's services options, rest options, graphql options, and server options. I named it `serverOptions` because I didn't want it to be confusing that it's not a `server` that you apass in. although I guess I don't see why it couldn't be.

Reply:
this is just semantics) .nevernimd